2023/2023-ssl-ball-placement-rules.html

```diff
@@ -0,0 +1,874 @@
+<!DOCTYPE html>
```

Comment:
This file does not belong to GIT. It will be build by the pipeline (once I've fixed the permission issues :see_no_evil: )